guilford_dps_offenders.yaml

```diff
@@ -0,0 +1,330 @@
+#psql :  offender_profile
+# intellij: "select * from offender_profile;"
+
+
+entityDefinitions:
+  offender:
+    fqn: "general.person"
+    entitySetName: "GuilfordOffenders"
+    propertyDefinitions:
+      nc.SubjectIdentification:
+        type: "nc.SubjectIdentification"
+        column: "OFFENDER NC DOC ID NUMBER"
+      nc.PersonSex:
+        type: "nc.PersonSex"
+        column: "OFFENDER GENDER CODE"
+        transforms:
+        - !<transforms.ReplaceTransform>
+          target: ["MALE", "FEMALE"]
+          goal: ["M", "F"]
+      nc.PersonBirthDate:
+        type: "nc.PersonBirthDate"
+        column: "OFFENDER BIRTH DATE"
+        transforms:
+        - !<transforms.DateTransform>
+          pattern: ["yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+      nc.PersonRace:
+        type: "nc.PersonRace"
+        # column: "OFFENDER RACE CODE"
```

Comment:
please delete this commented out line - we'll strive for a clean as possible yaml after we figure out what code works.